### PR TITLE
PEP8 fixes for test scripts

### PR DESCRIPTION
**Pull request by souravsingh**

### CODE REVIEW

File: gensim/corpora/bleicorpus.py [hunk 1]
```diff
@@ -109,14 +112,25 @@ def save_corpus(fname, corpus, id2word=None, metadata=False):
                 doc = list(doc)
                 offsets.append(fout.tell())
                 parts = ["%i:%g" % p for p in doc if abs(p[1]) > 1e-7]
-                fout.write(utils.to_utf8("%i %s\n" % (len(doc), ' '.join(parts))))
+                fout.write(
```
Comment (piskvorky) [inline gensim/corpora/bleicorpus.py]: Undesired change: makes code harder to read. Original better.

File: gensim/corpora/bleicorpus.py [hunk 2]
```diff
@@ -109,14 +112,25 @@ def save_corpus(fname, corpus, id2word=None, metadata=False):
...
+                    utils.to_utf8(
+                        "%i %s\n" %
+                        (len(doc), ' '.join(parts))))
 
-        # write out vocabulary, in a format compatible with Blei's topics.py script
+        # write out vocabulary, in a format compatible with Blei's topics.py
```
Comment (piskvorky) [inline gensim/corpora/bleicorpus.py]: dtto.

File: gensim/corpora/bleicorpus.py [hunk 3]
```diff
@@ -109,14 +112,25 @@ def save_corpus(fname, corpus, id2word=None, metadata=False):
...
+        # script
         fname_vocab = utils.smart_extension(fname, '.vocab')
-        logger.info("saving vocabulary of %i words to %s" % (num_terms, fname_vocab))
+        logger.info(
```
Comment (piskvorky) [inline gensim/corpora/bleicorpus.py]: Dtto.

Also, if fixing old code, can you replace `logger` methods which have `method("string" % params)` with `method("string", params)`?

File: gensim/corpora/bleicorpus.py [hunk 4]
```diff
@@ -109,14 +112,25 @@ def save_corpus(fname, corpus, id2word=None, metadata=False):
...
+            "saving vocabulary of %i words to %s" %
+            (num_terms, fname_vocab))
         with utils.smart_open(fname_vocab, 'wb') as fout:
             for featureid in xrange(num_terms):
-                fout.write(utils.to_utf8("%s\n" % id2word.get(featureid, '---')))
+                fout.write(
```
Comment (piskvorky) [inline gensim/corpora/bleicorpus.py]: Definitely not.

File: gensim/corpora/dictionary.py [hunk 5]
```diff
@@ -147,20 +153,23 @@ def doc2bow(self, document, allow_update=False, return_missing=False):
 
         token2id = self.token2id
         if allow_update or return_missing:
-            missing = dict((w, freq) for w, freq in iteritems(counter) if w not in token2id)
+            missing = dict((w, freq)
```
Comment (piskvorky) [inline gensim/corpora/dictionary.py]: No vertical indent in gensim. Use hanging indent.

This is the last comment but many instances of the same issues (unwanted line breaks, vertical indent) elsewhere in the proposed changes.

**Comment by souravsingh** [comment]
@piskvorky I am thinking of closing this PR and doing PEP8 fixes for individual modules. It would allow is to review the PR easier.

**Comment by tmylk** [comment]
Ping @souravsingh

**Pull request closed without merging**